tests/acceptance/features/coreApiSharePublicLink3/allowGroupToCreatePublicLinks.feature

```diff
@@ -1,4 +1,4 @@
-@api @files_sharing-app-required @notToImplementOnOcis
+@api @files_sharing-app-required @toImplementOnOCIS @issue-1328
 Feature: public share sharers groups setting
```

Comment:
I think this feature will be implemented differently in oCIS, so could be deleted.
CC @phil-davis